src/CommandingOfficers/Commander.java

```diff
@@ -125,12 +126,21 @@ protected void addCommanderAbility(CommanderAbility ca)
   }
 
   /**
-   * Allows a Commander to inject modifications before evaluating a battle.
+   * These functions Allow a Commander to inject modifications before evaluating a battle.
```

Comment:
FYI, Javadoc comments only apply to the immediately-following function. The comments describing `changeCombatContext` should be moved down to there.